spec/lcm/integration/shared_contexts_for_lcm_specs.rb

```diff
@@ -0,0 +1,264 @@
+require_relative 'support/configuration_helper'
+require_relative 'support/project_helper'
+require_relative 'support/connection_helper'
+require_relative 'support/lcm_helper'
+require 'gooddata_datawarehouse'
+
+def create_suffix
+  hostname = Socket.gethostname
+  timestamp = DateTime.now.strftime('%Y%m%d%H%M%S')
+  suffix = "#{hostname}_#{timestamp}"
+  segment_name_forbidden_chars = /[^a-zA-Z0-9_\\-]+/
+  suffix.scan(segment_name_forbidden_chars).each do |forbidden_characters|
+    suffix.gsub!(forbidden_characters, '_')
+  end
+  suffix
+end
+
+def create_workspace_csv(workspaces, client_id_column)
```

Comment:
Identical code found in shared_contexts_for_lcm_specs.rb:18, lcm_helper.rb:27 (mass = 156)